web/static/js/components/alert.jsx

```diff
@@ -10,6 +10,10 @@ Modal.defaultStyles.content.zIndex = 2
 Modal.defaultStyles.overlay.zIndex = 2
 
 export class Alert extends Component {
+  componentWillMount() {
+    Modal.setAppElement("body")
```

Comment:
@tnewell5 I'm confused about this. Should the element by the root node of the react application, rather than the body tag?